spec/factories/support/random_helpers.rb

```diff
@@ -0,0 +1,23 @@
+module RandomHelpers
+  def factory_rand(range)
+    if Rails.env.test?
+      range.min
+    else
+      rand(range)
+    end
+  end
+
+  def factory_sample(things, num = 1)
+    if Rails.env.test?
+      num == 1 ? things.first : things[0...num]
```

Comment:
```ruby
things.take(num)
```

Reply:
Will still need to check for `1` since `sample` returns an item for 1 and an array for anything else, but I can use it for the second part.